WordPress/build.gradle

```diff
@@ -171,7 +171,7 @@ android {
             buildConfigField "String", "TRACKS_EVENT_PREFIX", '"wpandroid_"'
             buildConfigField "String", "PUSH_NOTIFICATIONS_APP_KEY", '"org.wordpress.android"'
             buildConfigField "boolean", "ENABLE_QRCODE_AUTH_FLOW", "false"
-            buildConfigField "boolean", "ENABLE_OPEN_WEB_LINKS_WITH_JP_FLOW", "false"
+            buildConfigField "boolean", "ENABLE_OPEN_WEB_LINKS_WITH_JP_FLOW", "true"
```

Comment:
The true/false config for ENABLE_OPEN_WEB_LINKS_WITH_JP_FLOW as here makes more sense to me with respect the [previous PR](https://github.com/wordpress-mobile/WordPress-Android/pull/17497) 👍 . Maybe I'm just missing something, but just wanted to share I was expecting the "open web links with jetpack" feature flag in Me -> App Settings -> Debug to be `by default` true in WP and false in JP. Instead, I'm getting the flag always set to true after a fresh install on both apps 🤔 (thanks for letting me better understand if it's something obvious 🙇 )

Reply:
Just reporting what we were saying on slack and clarified what I was reporting above. The Debug settings list (that is what made me doubt it) is using the default value of the flag and not the isEnabled flag...that's why it is always reported as active in that screen...the overwritten isEnabled can actually add any further custom logic to make it finally active or not in the logic.